src/FormBuilder/resources/js/form-builder/src/blocks/fields/terms-and-conditions/Edit.tsx

```diff
@@ -234,7 +234,12 @@ function CheckboxPlaceholder({label, linkText, isFormDisplay, agreementText}: Ch
                     border: 'none',
                 }}
             >
-                <CheckboxControl label={label} onChange={null} disabled={true} />
+                <CheckboxControl
+                    className={'give-terms-and-conditions-control'}
+                    label={label}
```

Comment:
let's make sure to use the `givewp` prefix instead of `give`.

```suggestion
                    className={'givewp-terms-and-conditions-control'}
                    label={label}
```

Reply:
Will do!
Resolved: 5b961dbf78f36fc6c56253b7212a7608d3d3b6d1